experimenter/experimenter/nimbus-ui/src/components/AppLayoutWithSidebar/index.tsx

```diff
@@ -127,6 +128,21 @@ export const AppLayoutWithSidebar = ({
                 </LinkNav>
               ))}
 
+              <p className="edit-divider position-relative small my-2">
+                <span className="position-relative bg-light pl-1 pr-2 text-muted">
+                  History
+                </span>
+              </p>
+
+              <LinkNav
+                route={`/history/${slug}`}
+                testid={`history-page-${slug}`}
+                isExternalRoute={true}
```

Comment:
My preference is to use a more consistent URL (see previous comment), but if there is a good reason to keep the current URL, then I'd suggest renaming this prop. To me, "external" implies "outside of Experimenter". Maybe something like `relativeToRoot`?